src/constants.ts

```diff
@@ -1,8 +1,9 @@
 import { TMat2D } from "./typedefs";
 
 export { version as VERSION } from '../package.json';
-export const noop = () => {};
+export function noop() {};
```

Comment:
There is a difference that a fat arrow can't be bound and can't be used as a class member.
So since noop is used in createClass as an empty method, just to be sure i made it a function